File: 1-easy/problems/2717-semi-ordered-permutation/solution.py
from typing import List, Optional, Dict, Set, Tuple

class Solution:
    def semiOrderedPermutation(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        i, j = nums.index(1), nums.index(len(nums))
        return i+((len(nums)-1)-j)-int(i > j)

File: 1-easy/problems/2717-semi-ordered-permutation/solution.ts
function semiOrderedPermutation(nums: any): boolean | number | string | any {
  """
  :type nums: List[int]
  :rtype: int
  """
  i, j = nums.index(1), nums.index((nums).length)
  return i+(((nums).length-1)-j)-int(i > j);
}

export { semiOrderedPermutation };
